src/main/cpp/t10_unique.cpp

```diff
@@ -25,5 +25,21 @@
 using namespace std;
 
 int t10_unique() {
-
+ int n,i,k;
+ cin >> n;
+ int a[n];
+ for( i = 0; i < n ; i++) {
+     cin >> a[i];
+ }
+ for ( i = 0 ; i < n ; i++) {
+    bool l = 1;
+        for( k = 0; k < n ; k++) {
+            if( a[i] == a[k] & i != k) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean result is used in bitwise operation. Clarify expression with parentheses.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513127)